observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1-1.adoc

```diff
@@ -52,6 +36,13 @@ There are currently known issues:
 [id="distributed-tracing-rn_3-1-1_jaeger-release-notes"]
 == {JaegerName}
 
+The {JaegerName} is provided through the {JaegerOperator} Operator.
```

Comment:
It looks as if something isn't working with the {JaegerOperator} attribute here https://73592--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1-1#distributed-tracing-rn_3-1-1_jaeger-release-notes

Reply:
Thank you. Looks like a lost change. Fixed.